solutions/tier4/ids/endpoint/endpoint.yaml

```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #########
-# AC-17(1), SC-7(9), SC-18(1), SI-3(2), SI-3(4), SI-3(7), SI-4(4), SI-4(5), SI-4(7) - Defining Cloud IDS endpoint to receive mirrored traffic that performs threat detection and analysis
```

Comment:
All security control comments should remain. Is there a reason you'd like to remove this?